crate-features/Cargo.toml

```diff
@@ -18,8 +18,8 @@ lazy_static = { version = "1.4.0", features = ["spin", "spin_no_std"] }
 libc = { version = "0.2.62", features = ["extra_traits"] }
 rand_chacha = { version = "0.1.1" }
 regex-syntax = { version = "0.6.12" }
+reqwest = { version = "0.9.20", features = ["rustls-tls"] }
```

Comment:
@mvines , should `default-features = false` be included here?

Reply:
yeah I think so, the crate features need to be the same everywhere or rebuild hell will return